databricks/koalas/indexing.py

```diff
@@ -492,21 +494,35 @@ def __setitem__(self, key, value):
         rows_sel, cols_sel = key
 
         if (not isinstance(rows_sel, slice)) or (rows_sel != slice(None)):
-            raise SparkPandasNotImplementedError(
-                description="""Can only assign value to the whole dataframe, the row index
-                has to be `slice(None)` or `:`""",
-                pandas_function=".loc[..., ...] = ...",
-                spark_target_function="withColumn, select")
-
-        if not isinstance(cols_sel, str):
-            raise ValueError("""only column names can be assigned""")
+            if isinstance(cols_sel, list):
+                kdf = self._kdf
+                is_start = True
+                for col_sel in cols_sel:
+                    if is_start:
+                        sdf = kdf._sdf.withColumn(
+                            col_sel,
+                            (F.when(F.col(kdf._internal.index_columns[0]).isin(rows_sel), value)
+                              .otherwise(F.col(col_sel))))
+                        is_start = False
+                    else:
+                        sdf = sdf.withColumn(
+                            col_sel,
+                            (F.when(F.col(kdf._internal.index_columns[0]).isin(rows_sel), value)
+                              .otherwise(F.col(col_sel))))
+                self._kdf._internal = self._kdf._internal.copy(sdf=sdf)
+            else:
+                raise SparkPandasNotImplementedError(
+                    description="""Can only assign value to the whole dataframe, the row index
+                    has to be `slice(None)` or `:`""",
+                    pandas_function=".loc[..., ...] = ...",
+                    spark_target_function="withColumn, select")
 
         if isinstance(value, DataFrame):
             if len(value.columns) == 1:
                 self._kdf[cols_sel] = _col(value)
             else:
                 raise ValueError("Only a dataframe with one column can be assigned")
-        else:
+        elif not isinstance(cols_sel, list):
```

Comment:
What happens if `isinstance(cols_sel, list)`?

Reply:
Raises AssertionError because that logic already handled above `497 ~ 511`

Maybe this line removed, below will be happend

```python
>>> kdf.loc[['cobra', 'viper'], ['max_speed']] = 100
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/haejoonlee/Desktop/git_store/koalas/databricks/koalas/indexing.py", line 527, in __setitem__
    self._kdf[cols_sel] = value
  File "/Users/haejoonlee/Desktop/git_store/koalas/databricks/koalas/frame.py", line 7315, in __setitem__
    assert isinstance(value, DataFrame)
AssertionError
>>>
>>> kdf
            max_speed  shield
cobra             100       2
viper             100       5
sidewinder          7       8
>>>
```

although `max_speed` of `cobra` and `viper` changed well, it raise AssertionError.

Reply:
What if `kdf.loc[:, ['max_speed']] = 100`?

Reply:
oops i think i missed that case... i'm going to rearrange the logic and add this example to test case. Thanks!!